Source/WebCore/platform/graphics/GeometryUtilities.cpp

```diff
@@ -265,9 +265,8 @@ float toRelatedAcuteAngle(float angle)
     angle = toPositiveAngle(angle);
     if (angle < 90)
         return angle;
-    if (angle > 90 || angle < 180)
-        return std::abs(180 - angle);
-    return std::abs(360 - angle);
+    // FIXME: We likely don't handle an angle between 180 and 360 degrees correctly.
+    return std::abs(180 - angle);
```

Comment:
Doesn't this change the path for an angle that is exactly 90? Before it would return 270 and now 90.

Reply:
that would creating the mapping:
```
angle < 90: f(angle) => angle ∈ [0, 90]
90 <= angle <= 270: f(angle) => |angle - 180| ∈ [0, 90]
270 <= angle  <= 360: f(angle) => 360 - angle ∈ [0, 90]
```
these are all acute angles

Reply:
We need to write a unit test for this.

Maybe remove this from this PR and file a follow-up bug?

Reply:
ok. Added a FIXME referring to webkit.org/b/298890.